src/go/networking/cmd/proxy/wsl_integration_linux.go

```diff
@@ -91,5 +91,7 @@ func setupLogging(logFile string) {
 
 	if debug {
 		logrus.SetLevel(logrus.DebugLevel)
+	} else {
+		logrus.SetLevel(logrus.InfoLevel)
```

Comment:
Do we not need to change the caller to say `-debug=true/false` instead of `-debug true/false`?
https://github.com/rancher-sandbox/rancher-desktop/blob/f1260fe04a8fda097701027ab9c3193f6f2f76bf/pkg/rancher-desktop/backend/wsl.ts#L456

https://pkg.go.dev/github.com/spf13/pflag#readme-command-line-flag-syntax

Reply:
The docs link you attached is for `spf13` pkg; the `wsl-proxy` uses the built-in flag. I think it should be fine and the Go flag package should correctly parse `-debug true`.

Reply:
Ah, so actually after testing it, it looks like If a boolean flag like -debug is provided without an =, the Go flag package sets it to true immediately, and the next argument (false) is treated as a positional argument, not a value for `-debug` and false is treated as a stray argument. 🤯

Reply:
Ah, yeah, sorry about that; the matching docs is [here](https://pkg.go.dev/flag#hdr-Command_line_flag_syntax) which has the same behaviour. It has an explanation on why it's that way.